zcash_client_sqlite/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ and this library adheres to Rust's notion of
 [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [0.8.0-rc.1] - 2023-09-08
```

Comment:
Need the version bumps in the `Changed` section here also.

Reply:
They are?

Reply:
`zcash_proofs`

Reply:
Oh, I see I missed one: `zcash_proofs` is not a public dependency, so it shouldn't be in the "Changed" section.

Reply:
Oh... I guess that dependency is indirect?